test/typedarray/reflect_defineProperty.js

```diff
@@ -0,0 +1,104 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft Corporation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+const sample = new Float64Array(2)
+var result = null;
+
+// false: writable false
+result = Reflect.defineProperty(sample, "0", {
+    value: 42,
+    configurable: false,
+    enumerable: true,
+    writable: false,
+});
+console.log(sample[0]);

```

Comment:
We have lots of existing tests that use baseline files, but I believe we're trying to prefer assertion-style tests where possible, because it's easier to read when the code and the expected output is together in one file. Take a look at test/typedarray/delete.js as an example (there are many examples, I just chose one at random). It loads `UnitTestFramework.js` at the top, which provides the `assert` object used in the test cases. It invokes the tests with `testRunner.runTests`, and doesn't provide a baseline file.